modules/alphamat/tutorials/alphamat_tutorial.markdown

```diff
@@ -0,0 +1,53 @@
+Information Flow Alpha Matting {#tutorial_alphamat}
+============================
+
+This project was part of Google Summer of Code 2019.
+
+*Student:* Muskaan Kularia
+
+*Mentor:* Sunita Nayak
+
+Alphamatting is the problem of extracting the foreground from an image. The extracted foreground can be used for further operations like changing the background in an image.
+
+Given an input image and its corresponding trimap, we try to extract the foreground from the background. Following is an example:
+
+Input Image: ![](images/net_input_image.jpg)
+Input Trimap: ![](images/net_trimap.jpg)
+Output alpha Matte: ![](images/net_result.jpg)
+
+This project is implementation of "[[Designing Effective Inter-Pixel Information Flow for Natural Image Matting](http://people.inf.ethz.ch/aksoyy/ifm/)]" by Yağız Aksoy, Tunç Ozan Aydın and Marc Pollefeys[1]. It required implementation of parts of other papers [2,3,4].
+
+# Building
+
+This module uses the Eigen package.
+
+Build the sample code of the alphamat module using the following cmake command, followed by  ```make``` command.
```

Comment:
> \`\`\`make\`\`\`

Use single quotes for "inline" code. Triple quotes are used for multi-line code blocks only.

Reply:
Done